inc/RestCommand.php

```diff
@@ -61,9 +61,15 @@ public function set_auth( $auth ) {
 	public function create_item( $args, $assoc_args ) {
 		list( $status, $body ) = $this->do_request( 'POST', $this->get_base_route(), $assoc_args );
 		if ( Utils\get_flag_value( $assoc_args, 'porcelain' ) ) {
-			WP_CLI::line( $body['id'] );
+			if ( $status < 400 ) {
```

Comment:
error codes now set in exit code and/or error text. Put error code at end of line without trailing period to make it easier for parsing.